packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.spec.tsx

```diff
@@ -68,7 +68,9 @@ describe('CustomViewLoader', () => {
     expect(iFrame.getAttribute('id')).toBe(
       `custom-view-${TEST_CUSTOM_VIEW.id}`
     );
-    expect(iFrame.getAttribute('src')).toBe(TEST_CUSTOM_VIEW.url);
+    expect(iFrame.getAttribute('src')).toContain(
+      `/no-project/custom-view/${TEST_CUSTOM_VIEW.id}`
```

Comment:
The expected URL for the iFrame is:
`<MC_HOST>/<project_key>/custom-view/<custom_view_id>`